src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -232,12 +236,43 @@ const en = ({ isDivorce, partner, userCase }: CommonContent, alternativeServiceT
     userCase.state === State.AwaitingAmendedApplication
       ? 'Latest information'
       : `${userCase.state === State.AwaitingClarification ? 'What you need to do now' : 'Latest update'}`,
+  informationRequested: {
+    line1: `The court has reviewed your application for ${
+      isDivorce ? 'divorce' : 'dissolution'
+    }. You need to provide some additional information before your application can progress.`,
+    line2: 'We have sent you an email with the information the court needs.',
+    line3: 'What you need to do next',
+    line4: 'Read the court’s reasons for stopping the application and provide the requested information.',
+    line5: 'If documents have been requested, you will be able to upload them to the court when you respond.',
+    buttonText: 'Provide information',
+    buttonLink: RESPOND_TO_COURT_FEEDBACK,
+    line6: 'We will let you know once we have reviewed the information you provided.',
+  },
+  respondedToRequestForInformation: {
+    line1: 'You have responded to the court.',
+    line2: `Your application will be checked by court staff. You will receive an email notification by ${dateOfCourtReplyToRequestForInformationResponse} confirming whether it has been accepted. Check your junk or spam email folder.`,
+    line3: `Your ${partner} will then be sent a copy of the application. They will be asked to check the information and respond. If they do not respond then you will be told what you can do next to progress the application.`,
+    line4: `If you want to ‘serve’ (send) the documents to your ${partner} yourself then phone 0300 303 0642 to request it. Otherwise the court will do it.`,
+    line5:
+      'If you want the court to serve (send) the application to be served by post instead of by email, then phone 0300 303 0642.',
```

Comment:
There's a variable in common content that we could use for the phone number `0300 303 0642`